woocommerce/woocommerce-admin
# Refactor page handler API

Fixes #1432.

This PR introduces a new API for registering new React-powered pages and connecting PHP-powered pages to WooCommerce Admin.

It also introduces developer documentation for using the new API. See: `docs/page-controller.md`.

### Detailed test instructions:

- Open WooCommerce Admin
- Verify all WP admin menu items are present and link to the proper pages
- For each page: verify the breadcrumb titles and links
- Read the new documentation page
